foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -90,65 +140,21 @@ def queue_downloader_jobs(self, experiment: Experiment):
                 downloaded_urls.append(original_file.source_url)
 
             try:
-                logger.info("Queuing downloader job for URL: " + original_file.source_url,
+                logger.info("Queuing downloader job.",
                             survey_job=self.survey_job.id,
-                            downloader_job=downloader_job.id)
-                send_job(downloader_job.downloader_task, downloader_job.id)
+                            downloader_job=downloader_job.id,
+                            downloaded_urls=downloaded_urls)
+                message_queue.send_job(downloader_task, downloader_job.id)
             except Exception as e:
                 # If the task doesn't get sent we don't want the
                 # downloader_job to be left floating
-                logger.exception("Failed to enqueue downloader job for URL: "
-                                 + original_file.source_url,
+                logger.exception("Failed to enqueue downloader job.",
```

Comment:
Can we check how this logger.exception field works? I'm not actually sure that these ever appear in our nomad logs.

Reply:
They appear as log level ERROR.